spark_client/services.py

```diff
@@ -226,7 +226,9 @@ def create(
                         --role view in the command
         """
 
-        formatted_extra_args = " ".join([f"--{k}={v}" for k, v in extra_args.items()])
+        formatted_extra_args = " ".join(
+            [f"--{k}={v}" for k, values in extra_args.items() for v in values]
```

Comment:
I would probably allow people to also feed normal values, rather than lists (e.g. documentation should be updated otherwise). But what about we do:

```
def listify(value: Any) -> List[str]:
    return [str(v) for v in values] if isinstace(value, list) else [str(value)] 
[f"--{k}={v}" for k, values in extra_args.items() for v in listify(values)]
```

We should anyway update doc string to clarify that lists will be treated

Reply:
Sure, this helps retain the old interface as well which was nicer than having to pass everything as a list. Thanks Enrico.